frontend/pages/admin/IntegrationsPage/cards/IdentityProviders/components/GoogleWorkspaceSection/_styles.scss

```
@@ -0,0 +1,28 @@
+      // ~50px wider than the section, overflowing past the InfoBanner above.
+      // border-box makes its total width equal the section width, like the banner.
+      box-sizing: border-box;
+
+      // Stack the fields with consistent vertical spacing so each field's help
+      // text doesn't crowd the next field's label.
+      @include vertical-form-layout;
+    }
+  }
+
+  &__api-key-json {
+    font-family: "SourceCodePro", $monospace;
```

Stylelint (17.13.0)

[error] Expected no quotes around "SourceCodePro" (font-family-name-quotes)

(font-family-name-quotes)